client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -149,7 +149,7 @@ public Object invoke(Object proxy, Method method, Object[] arguments) {
         final Observable<HttpClientResponse<ByteBuf>> resp = fullReq
             .submit(rxClient)
             .timeout(timeout, timeoutUnit)
-            .switchIfEmpty(logEmptyResponse(fullReq));
+            .switchIfEmpty(retry(fullReq, method, arguments, rxClient, true));
```

Comment:
Is there a point in making the request from scratch or would it have worked with `.submit().single().retry(3)`?

Reply:
Doing this will bubble up the sequence contains no elements exception to the other framework retry handling exceptions and it will try config.retryCount() x config.retryCount() times.

Reply:
ok, but Isn't that a good thing? That you have only one retry mechanism?

Reply:
I see where your aiming. Good point. Will try to fix tomorrow.